lib/choicesDropdown.js

```diff
@@ -530,18 +530,18 @@ export default class ChoicesDropdown {
    * */
   createOptionNode (value, key) {
     const optionNode = document.createElement('div')
-    optionNode.innerHTML = `<span>${value}</span><span icon></span>`
+    optionNode.innerHTML = `<span>${value}</span>`
     addClass(optionNode, ACTIVE_CLASS_NAMES.optionsItem.join(' '))
     optionNode.setAttribute(KEY_ATTRIBUTE_NAME, key)
 
-    optionNode.addEventListener('mouseenter', ({ target }) => {
-      this.hoverOption(target)
-    })
-
     optionNode.addEventListener('click', () => {
       this.addValue(value, key, true)
     })
 
+    // optionNode.addEventListener('mouseenter', ({ target }) => {
```

Comment:
?

Reply:
I've done some research and most of the dropdown libraries **don't** select suggestions on mouseenter. Suggestions can be selected only using the keyboard. Even the mt-select component follows the same behavior. Therefore, this feature is unnecessary and, on my opinion, should be deprecated.

Reply:
It's kind of confusing. By "select" I mean hover.

Reply:
sure, remove it fully from the code then :)